### PR TITLE
fix(connectors): prefer wrappers over bind

### PR DESCRIPTION
**Pull request by bobylito**
Previously we were using bind, which will be applied at each init. This
is ok for one time mount, but since we can mount and unmount dynamically
now, it's a problem.

### CODE REVIEW

**Comment by Haroenv** [comment]
Hey 👋 looks like a good call. You still need to run prettier on this PR :)

**Comment by bobylito** [comment]
> Hey 👋 looks like a good call. You still need to run prettier on this PR :)

Well the only problem is that I haven't touched the files failing...

**Comment by bobylito** [comment]
I need to fix the tests here :/

File: src/connectors/price-ranges/connectPriceRanges.js [hunk 1]
```diff
@@ -158,13 +158,15 @@ export default function connectPriceRanges(renderFn, unmountFn) {
       },
 
       init({ helper, instantSearchInstance }) {
-        this._refine = this._refine.bind(this, helper);
+        this.refine = (...opts) => {
+          this._refine(helper, ...opts);
```
Comment (Haroenv) [inline src/connectors/price-ranges/connectPriceRanges.js]: refine only has two arguments, what about not spreading?

Comment (bobylito) [reply src/connectors/price-ranges/connectPriceRanges.js]: That's indeed preferable to avoid spreading if we know the number of arguments 👍

**Comment by bobylito** [comment]
Happy to take 5 minutes and go over the code to explain if you want @Haroenv

**Pull request merged**